Jenkinsfile-Release

```diff
@@ -12,22 +12,57 @@
  */
 @Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
 
-def jira = [
+final jira = [
     versionPrefix: '', project: 'INT', projectId: '12410',
     credentialId : 'jenkins-jira', autoRelease: true, failOnError: true
 ]
 
-def jiraVersionMappings = [
-    'nexus-iq-server':'helm-iq', 
-    'nexus-repository-manager':'helm-nxrm'
+final jiraVersionMappings = [
+  'nexus-iq-server': 'helm-iq',
+  'nexus-repository-manager': 'helm-nxrm',
 ]
 
+final chartLocation = [
+  'nexus-iq-server': 'nexus-iq',
+  'nexus-repository-manager': 'nexus-repository-manager',
+]
+
+properties([
+  parameters([
+    choice(
+      choices: ['', 'nexus-iq-server', 'nexus-repository-manager'],
```

Comment:
default the choice to '', so the user very intentionally needs to choose a chart, and won't accidentally try to upgrade nexus-iq-server to a nexus-repository-manager version.